src/compiler.js

```diff
@@ -3,6 +3,32 @@
 var BuildMail = require('buildmail');
 var libmime = require('libmime');
 
+// HACK(rtasson): the following couple of lines are hacks to permit us to
+// modify the MIME part boundary prefix without upgrading our nodemailer
+// version to one that supports this natively.
+// The following baseBoundaryPrefix is what nodemailer currently uses:
+// https://github.com/nodemailer/nodemailer/blob/9b5fb94767c6d9ba0851dc487b5b4a2842cdae75/lib/mime-node/index.js#L44
+let baseBoundaryPrefix = '--_NmP';
+
+/**
+ * A convenience function to allow us to change the baseBoundaryPrefix.
+ *
+ * @param {String} prefix The MIME boundary prefix to set
+ */
+modules.export.setBaseBoundaryPrefix = function(prefix) {
```

Comment:
```suggestion
module.exports.setBaseBoundaryPrefix = function(prefix) {
```

This also needs to be done after `module.exports = Compiler` so that it's not overwritten.

Reply:
I think `src/nodemailer.js` will also need to export this function so that it can be accessed outside of this package, since the entrypoint (as indicated by the `main` field in `package.json`) is `src/nodemailer.js` and you can't do `require('@mixmaxhq/nodemailer/../compiler)`.